feat: manage buffers payload builder

rakpawel opened the pull request:
Adds payload builder for adding and removing liquidity from liquidity buffers.

Xeonus commented on components/ManageBufferModule.tsx:
```diff
@@ -0,0 +1,696 @@
+    GetTokensDocument,
+    {
+      variables: {
+        chainIn: [selectedNetwork],
```
This is not correct. Currently passes "MAINNET" etc instead of integer chain ID -> therefore sims fail.

You can test buffer removal from this address: 0x0efccbb9e2c09ea29551879bd9da32362b32fc89 for waEthWETH on mainnet and do this until sim is successful

Xeonus commented on components/ManageBufferModule.tsx:
```diff
@@ -0,0 +1,696 @@
+                : "Min Underlying Token Out"}
+            </FormLabel>
+            <Input
+              name="underlyingTokenAmount"
```
There are some checks for boundary conditions later but how can we make sure the shares to remove amount is correct for underlying?

Xeonus commented on components/ManageBufferModule.tsx:
```diff
@@ -0,0 +1,696 @@
+  }, [
+    selectedNetwork,
+    selectedToken,
+    sharesAmount,
```
We need a general logic so that if one parameter like shares to remove is triggered, then the correct amount of tokens should be calculated and prefilled? Otherwise tx / sim will fail

Xeonus reviewed (changes requested):
- need fix on chainID parsing 
- need additional logic on how to handle share and underlying amounts

rakpawel commented:
> * need fix on chainID parsing
> * need additional logic on how to handle share and underlying amounts

Fixed chainId stuff, sorry about that.

For the additional logic, added calculating token amounts after changing the share amount with 0.5% slippage applied.

Adding liquidity expects maximum token amounts, removing liquidity expects minimum token amounts - so we could expect users to change these values themselves without failing the simulation.
Seems clearer to me, especially when add/remove needs to be a balanced operation.

https://github.com/user-attachments/assets/a465128b-bee9-4041-b6c1-6b7494e8a138

Xeonus reviewed (approved):
- correctly fetches balances
- test with kpk managed safe buffer successful
- no eoa support yet, but that is fine (can come later)
- new module adheres to current pattern used

Very nice job!
